crates/noirc_frontend/src/hir/mod.rs

```diff
@@ -107,22 +107,21 @@ impl Context {
         &self,
         crate_id: &CrateId,
         pattern: FunctionNameMatch,
-    ) -> Vec<(String, FuncId)> {
+    ) -> Vec<(String, FuncId, bool)> {
```

Comment:
See above. This tuple structure is growing and we should name things appropriately.